app/controllers/otus_controller.rb

```diff
@@ -12,8 +12,8 @@ def index
         render '/shared/data/all/index'
       end
       format.json {
-        @otus = Queries::Otu::Filter.new(filter_params).all
-          .where(project_id: sessions_current_project_id).page(params[:page]).per(500)
+        @otus = Queries::Otu::Filter.new(filter_params.merge({project_id: sessions_current_project_id})).all
```

Comment:
Not sure I like this pattern.  We need to come up with guidlines for whether filters should include project_id reference or not.